assets/fxses/canonball_hit.json

```diff
@@ -1 +1,51 @@
-{"local_coords":false,"emission_shape":"Point","one_shot":true,"lifetime":0.2,"lifetime_randomness":0,"explosiveness":0.65,"amount":41,"shape":{"Circle":{"subdivisions":10}},"emitting":false,"initial_direction":{"x":0,"y":-1},"initial_direction_spread":6.2831855,"initial_velocity":73.9,"initial_velocity_randomness":0.2,"linear_accel":0,"size":64.0,"size_randomness":0.0,"blend_mode":"Alpha","colors_curve":{"start":{"r":0.8200004,"g":1,"b":0.31818175,"a":1},"mid":{"r":0.71000004,"g":0.36210018,"b":0,"a":1},"end":{"r":0.02,"g":0,"b":0.000000007152557,"a":1}},"gravity":{"x":0,"y":0},"post_processing":{}}
+{
```

Comment:
I am not so sure of pretty-printed jsons here. Usually they are copy-pasted to/from the editor back and forth all the time, and pretty-printing on each commit would be some extra work?

Reply:
Oh, I found the tool generates settings in a slightly different format, such as blend_mode being an object instead of just a string. I formatted it to check for differences to the others, but I'll fix it!